script/strategies/aave-v3/BaseDeployStrategies.sol

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import { BaseDeployPeriphery } from "../../BaseDeployPeriphery.sol";
+
+import { IEarnVault } from "@balmy/earn-core/interfaces/IEarnVault.sol";
+import { IGlobalEarnRegistry } from "src/interfaces/IGlobalEarnRegistry.sol";
+import {
+  AaveV3StrategyFactory,
+  AaveV3Strategy,
+  IAaveV3Pool,
+  IAaveV3Rewards,
+  IAToken,
+  AaveV3StrategyData
+} from "src/strategies/instances/aave-v3/AaveV3StrategyFactory.sol";
+import { IEarnBalmyStrategy } from "src/interfaces/IEarnBalmyStrategy.sol";
+import { StrategyId } from "@balmy/earn-core/interfaces/IEarnStrategy.sol";
+import { console2 } from "forge-std/console2.sol";
+
+contract BaseDeployStrategies is BaseDeployPeriphery {
+  function deployAaveV3Strategy(
+    address aaveV3Pool,
+    address aaveV3Rewards,
+    IAToken aToken,
+    bytes32 tosGroup,
+    bytes32 signerGroup,
+    address[] memory guardians,
+    address[] memory judges,
+    bytes memory feesData
```

Comment:
Let's pass a `Fee` struct here. If all values are `max(uint16)`, then we set "". If not, we'll encode it. We should also create a `DEFAULT_FEES` constant somewhere that has all values in `max(uint16)`